app/views/layouts/application.html.erb

```diff
@@ -1,36 +1,31 @@
 <!DOCTYPE html>
-<html lang="en" class="container">
+<html lang="en">
   <head>
-    <title>OSS Friday</title>
+    <title>Open Source Friday</title>
     <%= csrf_meta_tags %>
 
     <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
     <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
-    <% if Rails.env.development? %>
+    <% if false && Rails.env.development? %>
     <script src="https://github.com/Khan/tota11y/releases/download/0.1.3/tota11y.min.js"></script>
     <% end %>
   </head>
-  <header class="header jumbotron text-center">
-    <h1>OSS Friday</h1>
-    <h2><%= @subtitle || "Contribute to open source every Friday" %></h2>
-    <div><%#= render :partial => 'layouts/auth_nav' %></div>
-  </header>
   <body>
+    <%= render 'layouts/nav' unless request.path == "/" %>
+
     <%= yield %>
-  </body>
-  <footer class="footer">
-    <nav class="navbar navbar-inverse">
-      <div class="container-fluid">
-        <div class="collapse navbar-collapse">
-        <ul class="nav navbar-nav navbar-left">
-          <li><a href="/contributors/">Contributors</a></li>
-          <li><a href="/maintainers/">Maintainers</a></li>
-          <li><a href="/companies/">Companies</a></li>
-        </ul>
-        <ul class="nav navbar-nav navbar-right">
-          <li><a href="/">Home</a></li>
-        </ul>
+
+    <footer class="border-top">
+      <div class="container-md text-center text-gray py-6 px-3">
+        <%= octicon 'code', height: 20, class: "v-align-middle fill-gray mr-1", "aria-label": "code" %>
```

Comment:
Probably?

Primer is really interesting because it puts so much of the styles in markup. The advantage is it makes it _really_ fast to build out new sites and iterate on designs. The downside is that the markup is…interesting.